submissions/develabr/oop/index.js

```diff
@@ -0,0 +1,81 @@
+class Creature {
+  constructor(species, name, gender, leg, hands, saying, friends) {
+    this.species = species;
+    this.name = name;
+    this.gender = gender;
+    this.leg = leg;
+    this.hands = hands;

```

Comment:
From the [requirements](https://kottans.org/documentation/docs/doc/code-review/#oop-exercise):
> Classes for species that do not have hands by natural design do not consequently have hands or any equivalent property

Subclasses inherit all properties from parent classes, which means they have it.